R/workflows.R

```diff
@@ -94,9 +96,11 @@ build_workflow <- function(root, reports, ref) {
 construct_workflow <- function(reports, report_names, dependencies) {
   dependencies_graph <- workflow_dependencies(report_names, dependencies)
   order <- topological_sort(dependencies_graph)
+  all_report_names <- vcapply(reports, function(report) report$name)
+
   build_item <- function(report_name) {
     ## There may be multiple reports due to be run with this name
-    report_details <- reports[report_name == report_names]
+    report_details <- reports[report_name == all_report_names]
```

Comment:
this line was the bug

Reply:
Oh I broke this in the last PR https://github.com/vimc/orderly.server/pull/88/files#diff-42a27934b7a35c32e509cbfb5944e521e5ddbb1912b2c10e4f81761c02a8e195L94 oops!